errai-data-binding/src/main/java/org/jboss/errai/databinding/rebind/BindableProxyGenerator.java

```diff
@@ -253,13 +258,22 @@ private void generateGetter(final ClassStructureBuilder<?> classBuilder, final S
     }
   }
 
+  private boolean isToBeIgnored(final MetaMethod method) {
+    return method != null && method.isAnnotationPresent(IgnoreBinding.class);
+  }
+
   /**
    * Generates a setter method for the provided property plus the corresponding code for the
    * implementation of {@link HasProperties#set(String, Object)}.
    */
   private void generateSetter(final ClassStructureBuilder<?> classBuilder, final String property, final BlockBuilder<?> setMethod) {
     final MetaMethod getterMethod = bindable.getBeanDescriptor().getReadMethodForProperty(property);
     final MetaMethod setterMethod = bindable.getBeanDescriptor().getWriteMethodForProperty(property);
+    
+    if(isToBeIgnored(setterMethod)){
```

Comment:
Please see comment in `generateGetter`. If that check is moved it would make sense to move this one as well.